scripts/setup_byoid.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+# Copyright 2021 Google LLC.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+# This file is a mostly common setup file to ensure all BYOID integration tests
+# are set up in a consistent fashion across the languages in our various client libraries.
+# It assumes that the current user has the relevant permissions to run each of
+# the commands listed.
+
+suffix=""
+
+function generate_random_string () {
+  local valid_chars=abcdefghijklmnopqrstuvwxyz0123456789
+  for i in {1..8} ; do
+    suffix+="${valid_chars:RANDOM%${#valid_chars}:1}"
+    done
```

Comment:
Why not keep suffix as a local value and `echo "$suffix"` in this function.

And then below:
`suffix=$(generate_random_string)`